CodingStyle.cc

```diff
@@ -0,0 +1,112 @@
+/*=====================================================================
+ 
+ QGroundControl Open Source Ground Control Station
+ 
+ (c) 2009 - 2015 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+ 
```

Comment:
Just because we are discussing this, most files have some old date and a lot of them don't have this header at all.